app/views/new_document/choose_supertype.html.erb

```diff
@@ -15,7 +15,6 @@
         items: Supertype.all.map do |supertype|
           id = supertype.id
           label = I18n.t("supertypes.#{id}.label")
```

Comment:
Can this also be switched to just `t` so that it raises on a missing translation?